.swiftlint.yml

```diff
@@ -3,6 +3,7 @@ excluded:
   - Pods
   - Scripts
   - vendor
+  - DerivedData
```

Comment:
Xcode uses a shared `DerivedData` data folder by default, but I set mine as local to each project.

A local `DerivedData` can be easily removed manually (`rm -rf ./DerivedData`) and it makes it easier to find build products and other stuff when one needs to inspect them.